tests/Operation/FindAndModifyFunctionalTest.php

```diff
@@ -92,6 +93,23 @@ public function testHintOptionAndUnacknowledgedWriteConcernUnsupportedClientSide
         $operation->execute($this->getPrimaryServer());
     }
 
+    public function testFindAndModifyReportedWriteConcernError(): void
+    {
+        try {
+            $operation = new FindAndModify(
+                $this->getDatabaseName(),
+                $this->getCollectionName(),
+                ['remove' => true, 'writeConcern' => new WriteConcern(50)]
+            );
+
+            $operation->execute($this->getPrimaryServer());
+        } catch (CommandException $e) {
+            $this->assertInstanceOf(CommandException::class, $e);
```

Comment:
This check is unnecessary - any class that isn't a descendant of `CommandException` won't have been caught here.

At the same time, we can simplify this by using the `expectException`, `expectExceptionMessage`, and `expectExceptionCode` methods of PHPUnit.

Reply:
I decide to use `expectExceptionMessageMatches` instead of `expectExceptionMessage` because I need to check that message contains `Write Concern Error` not all message. Is it ok ?